### PR TITLE
chore(yamls): synchronize cstor-operator yaml with a release version

### PR DESCRIPTION
**Pull request by mittachaitu**
This PR updates the cstor-operator.yaml to have
similar changes as [release version](https://github.com/openebs/charts/blob/gh-pages/cstor-operator.yaml) YAML

Signed-off-by: mittachaitu <sai.chaithanya@mayadata.io>

### CODE REVIEW

**Comment by mittachaitu** [comment]
Need to update `make manifests` to pull ndm-operator from openebs/charts repo and make modifications to fit for cStor like naming convention, serviceAccount e.t.c..

File: deploy/cstor-operator.yaml [hunk 1]
```diff
@@ -3886,195 +3840,214 @@ spec:
...
-        imagePullPolicy: IfNotPresent
-        env:
-        - name: NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
-        - name: METRICS_LISTEN_PORT
-          value: :9100
+        - name: ndm-cluster-exporter
+          image: openebs/node-disk-exporter:1.7.0
```
Comment (kmova) [inline deploy/cstor-operator.yaml]: By any chance.. is this YAML also used by the CI tests. In that case - we probably need this to be "ci".

Comment (prateekpandey14) [reply deploy/cstor-operator.yaml]: yes, we have to use `ci` tag

Comment (mittachaitu) [reply deploy/cstor-operator.yaml]: Updated to `ci` tag

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**